packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -675,99 +712,118 @@ - (NSDictionary *)encodingMap
 
 #pragma mark - Internal methods
 
-- (void)_downloadResumableCreateSessionWithUrl:(NSURL *)url withScopedPath:(NSString *)scopedPath withUUID:(NSString *)uuid withOptions:(NSDictionary *)options withResumeData:(NSData * _Nullable)resumeData withResolver:(UMPromiseResolveBlock)resolve withRejecter:(UMPromiseRejectBlock)reject
+- (EXFileSystemSessionType)_importSessionType:(NSNumber * _Nullable)type
 {
-  NSURLSessionConfiguration *sessionConfiguration = [NSURLSessionConfiguration defaultSessionConfiguration];
+  return [type intValue] ?: EXFileSystemBackgroundSession;
+}
+
+- (NSURLSession *)_sessionForType:(EXFileSystemSessionType)type
+{
+  switch (type) {
+    case EXFileSystemBackgroundSession:
+      return _backgroundSession;
+    case EXFileSystemForegroundSession:
+      return _foregroundSession;
+  }
+  return nil;
+}
+
+- (void)_unregisterIfResumableTaskDelegate:(EXSessionTaskDelegate *)exTask
```

Comment:
Could you move it closer to the section where it is called, so it's easier to compare call and its effects?